app/assets/stylesheets/application.scss

```diff
@@ -19,6 +20,7 @@
 // Variables
 $green: #D0E3CC;
```

Comment:
Can delete the green here that I had added

Reply:
Deleted